CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # TRILLIAN Changelog
 
+## v1.0.9 - CloudSpanner Improvements & Internal API changes etc.
+
+Published 2018-04-17 07:01:46 +0000 UTC
+
+Changes are in progress (e.g. see PR #1037) to rework the internal signed root format used by the log RPC server to be more useful / interoperable. Currently they are mostly internal API changes to the log and map servers.
+
+Changes have been made to log server APIs and CT frontends for when a request hits a server that has an earlier version of the tree than is needed to satisfy the request. In these cases the log server used to return an error but now returns an empty proof along with the current STH it has available. This allows clients to detect these cases and handle them appropriately.
+
+The CloudSpanner schema has changed. If you have a database instance you'll need to recreate it with the new schema. Performance has been noticeably improved since the previous release and we have tested it to approx one billion log entries. Note: This code is still being developed and further changes are possible.
+
+Support for `sqlite` has been removed because of ongoing issues with flaky tests. These were caused by concurrent accesses to the same database, which it doesn't support.
```

Comment:
Maybe "Support for `sqlite` in unit tests ...." to make it clearer that we've not removed anything important.

Reply:
Done.